metabase/metabase
# replace dc.js line/area/bar/combo charts with newer echarts implementation without interactivity

Closes https://github.com/metabase/metabase/issues/36183

### Description

Basic setup for dynamic combo chart without interactivity.

### How to verify

Open different line/area/bar/combo charts and ensure they look ok in QB and dashboards.

### Demo

Charts on video have slightly different styles because I unlinked zrender package locally

https://github.com/metabase/metabase/assets/14301985/f44df901-1089-41c2-b317-480fcef10e89

### Checklist
